sys/net/application_layer/dhcpv6/client.c

```diff
@@ -255,10 +255,12 @@ static inline size_t _compose_elapsed_time_opt(dhcpv6_opt_elapsed_time_t *time)
     return len + sizeof(dhcpv6_opt_t);
 }
 
-#ifdef MODULE_GNRC_DHCPV6_CLIENT_MUD_URL
 static inline size_t _compose_mud_url_opt(dhcpv6_opt_mud_url_t *mud_url_opt,
                                           size_t len_max)
 {
+#ifdef MODULE_GNRC_DHCPV6_CLIENT_MUD_URL
+    return 0;
```

Comment:
Looks like you have that reversed :wink:

Reply:
Oops, classic mistake :D